src/main/java/de/tum/in/www1/artemis/web/rest/ProgrammingExerciseResource.java

```diff
@@ -416,7 +418,7 @@ public ProgrammingExerciseResource(ProgrammingExerciseRepository programmingExer
      * @param notificationText to notify the student group about the updated problemStatement on the programming exercise
      * @return the ResponseEntity with status 200 (OK) and with body the updated problemStatement, with status 404 if the programmingExercise could not be found, or with 403 if the user does not have permissions to access the programming exercise.
      */
-    @PatchMapping("/programming-exercises-problem")
+    @PatchMapping(Endpoints.PROBLEM)
```

Comment:
Ideally we refactor this URL to something like 

/programming-exercises/{exerciseId}/problem-statement

Reply:
I refactored the endpoint and removed the now obsolete DTO `ProblemStatementUpdate`. I also updated the client and the related server test